packages/botonic-plugin-contentful/src/contentful/contents/button.ts

```diff
@@ -95,12 +96,14 @@ export class ButtonDelivery extends ContentDelivery {
   }
 
   // TODO move to a new CmsUtils.buttonToCallback(cms.ContentCallback)?
-  private static fromContentReference(
-    entry: contentful.Entry<CommonEntryFields>
+  private fromContentReference(
+    entry: contentful.Entry<CommonEntryFields>,
+    context: cms.Context
   ): cms.Button {
     const fields = entry.fields
     let text = fields.shortText
-    if (!text) {
+    const fixMissingData = context.fixMissingData ?? true
+    if (!text && fixMissingData) {
```

Comment:
@dpinol If `fixMissingData` is set to `false`, no message will be logged, is this the expected behaviour? Also, the Button will be created with an empty text, may this cause any errors?

Reply:
if the developer manually sets the flag to false, I don't see why we should log anything. Probably developers will only do it when they set empty data purposely, and then they don't want spam in logs.

>Button will be created with an empty text, may this cause any errors?
Yes, that might crash. I fixed it at  4c4812c391d203ba2c42d22f85dbf33aa21c6d8e on the lastest PR. thanks!